### DISCUSSION TITLE
ARM support is not really realistic the way it is

### DISCUSSION THREAD

**Discussion by lanmower** [wasp-lang/wasp#1441] [Archived]
I tried to run wasp, and because it wasn't compiled for arm, I couldn't run it, it also didn't want to compile, after needing to free over 11gb of space for cabal to learn it doesn't work.

This was a very negative user experience, it took an hour and it was unrewarding since it was impossible to get it running

**Comment by Martinsos** [comment]
Hey @lanmower , sorry to hear about that! 
Why did you expect Wasp to support ARM? Also, can you tell me a bit more about why you need support for ARM?

Cabal -> what stopped you there, and when you say you learned that it doesn't work, what exactly did you learn? That Cabal in general doesn't not support ARM?

I haven't dealt much with ARM recently so sharing as much information here as you can would be quite helpful!
